src/App.tsx

```diff
@@ -46,16 +111,22 @@ export const App: React.FC = () => {
         >
           Reset
         </button>
+        {(sortType !== SortType.NONE || isReversed === true)
+          && (
+            <button
+              type="button"
+              className="button is-danger is-light"
+              onClick={() => reset()}
+            >
+              Reset
+            </button>
+          )}
       </div>
 
       <ul>
         <ul>
-          <li data-cy="Good">Dumplings</li>
-          <li data-cy="Good">Carrot</li>
-          <li data-cy="Good">Eggs</li>
-          <li data-cy="Good">Ice cream</li>
-          <li data-cy="Good">Apple</li>
-          <li data-cy="Good">...</li>
+          {getReorderedGoods(goodsFromServer, { isReversed, sortType })
+            .map(good => <li data-cy="Good">{good}</li>)}
```

Comment:
```suggestion
          {preparedGoods
            .map(good => <li data-cy="Good">{good}</li>)}
```

create a constant in the component and assign the result of the sort function in it!